server/models/eve.py

```diff
@@ -41,6 +41,13 @@ def get_multi(cls, id_list):
         db.session.commit()
         return return_items
 
+def get_prices(types):
```

Comment:
`types` should be named `type_id_list` so it isn't ambiguous whether it's a list of `Type` objects or of `Type.id` values.